visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -126,9 +126,23 @@ export interface AppSettings {
 	isLoadingFile: boolean
 	sortingOrderAscending: boolean
 	searchPanelMode: SearchPanelMode
+	layoutAlgorithm: LayoutAlgorithm
+	maxTreeMapFiles: number
 	isAttributeSideBarVisible: boolean
 }
 
+export enum LayoutAlgorithm {
+	SquarifiedTreeMap = "Squarified TreeMap",
+	StreetMap = "StreetMap",
+	TMStreet = "TMStreet"
```

Comment:
This was the TreeMap Street, which combines Squarified and the Street right? I would then rather use some more descriptive name. Either `TreeMapStreet` or maybe you have more ideas?